docs/rules/no-catch-shadow.md

```diff
@@ -18,7 +18,7 @@ console.log(err)    // err is 'problem', not 'x'
 
 This rule is aimed at preventing unexpected behavior in your program that may arise from a bug in IE 8 and earlier, in which the catch clause parameter can leak into outer scopes. This rule will warn whenever it encounters a catch clause parameter that has the same name as a variable in an outer scope.
 
-The following patterns are considered problems:
+Examples of **incorrect** code if you turn on the rule:
```

Comment:
When discussing options, you've used the wording `Examples of incorrect code for the <x> option:`.  I wonder if we could use a similar wording here, like `Examples of incorrect code for this rule:`.  That would be a little more concise and readable, at least in my opinion.